enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1158 @@
+operational aspects are expected to be introduced.
+
+## Support Procedures
+
+Change management problems created by faulty implementations will need to be resolved by 
+analyzing operator logs. The operator responsible for a given resource will vary. Existing
+support tooling like must-gather should capture the information necessary to understand
+and fix issues. 
+
+Change management problems where user expectations are not being met are designed to
+be informed by the detailed `status` provided by the resources bearing the `changeManagement`
+stanza in their `spec`.
+
+## Alternatives
+
+### Implement maintenance schedules via an external control system (e.g. ACM)
+We do not have an offering in this space. ACM is presently devoted to cluster monitoring and does
+not participate in cluster lifecycle.
+
+### Do not separate control-plane and worker-node updates into separate phases
+As separating control-plane and worker-node updates into separate phases is an important motivation for this
+enhancement, we could abandon this strategic direction. Reasons motivating this separation are explained 
+in depth in the motivation section.
+
+### Separate control-plane and worker-node updates into separate phases, but do not implement the change control concept
+As explained in the motivation section, there is a concern that implementing this separation without
+maintenance schedules will double the perceived operational overhead of OpenShift updates. 
+
+This also creates work for our Service Delivery team without any platform support.
+
+### Separate control-plane and worker-node updates into separate phases, but implement a simpler MaintenanceSchedule strategy
+We could implement change control without `disabledUntil`, `pausedUntil`, `exclude`, and perhaps more. However,
+it is risky to impose a single opinionated workflow onto the wide variety of consumers of the platform. The workflows
+described in this enhancement are not intended to be exotic or contrived but situations in which flexibility
+in our configuration can achieve real world, reasonable goals. 
+
+`disabledUntil` is designed to support our Service Delivery team who, on occasion, will need
+to be able to bypass configured change controls. The feature is easy to use, does not require 
+deleting or restoring customer configuration (which may be error-prone), and can be safely 
+"forgotten" after being set to a date in the future.
+
+`pausedUntil`, among other interesting possibilities, offers a cluster lifecycle administrator the ability
+to stop a problematic update from unfolding further. You may have watched a 100 node
+cluster roll out a bad configuration change without knowing exactly how to stop the damage
+without causing further disruption. This is not a moment when you want to be figuring out how to format
+a date string, calculating timezones, or copying around cluster configuration so that you can restore
+it after you stop the bleeding.
```

Comment:
What is driving the request for this feature, managed services or users with self-managed clusters?

Reply:
It unfolded as follows (see motivation section for more depth on each point):
1. There is a negative perception of upgrades due to worker nodes being entangled with the end to end process. It also forces unnecessary maintenance on nodes and risks for customers.
2. We can eliminate those problems and empower users by separating the control-plane and worker-node updates.
3. @derekwaynecarr raised the concern that doing so would double operational burden. Also guidance that whatever improvements we make in standalone should be in HCP.
4. Maintenance windows were identified as a tool which could prevent new operational burden for customers who may not benefit from the separation. Maintenance windows are also a common configuration in other managed kubernetes solutions.
5. SRE, as part of all these conversations, indicates product level support for maintenance windows would help simplify their existing implementation. 
6. Exploration of all other benefits of change management & scheduling which simplify operations.

So, item (1) was the genesis (particularly in standalone since SRE manages to insulate customers from gaps today), but we found additional value along the path.